### ISSUE TITLE
Suggestion: redstone control for the quantum computer

### ISSUE THREAD

**Issue by walksanatora** [acikek/qcraft#1]
allow the quantum computer to swap when it receives a redstone signal instead of only the gui button

**Comment by acikek** [comment]
Good idea, unfortunately not able to work on this mod right now. MIght do it when updating to 1.19.
